integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -1020,4 +1021,22 @@ public static void verifyServerCommunication(String curlRequest, List<String> ma
         });
   }
 
+  /**
+   * Get a unique name.
+   * @param prefix prefix of the name
+   * @param suffix suffix of the name
+   * @return the full name
+   */
+  public static String getUniqueName(String prefix, String... suffix) {
```

Comment:
Why copy this method here? Or, put another way, why isn't aren't you using a method already in UniqueName or creating a variant in that class? This will lead to code duplication if other classes need a unique name with a prefix and multiple suffixes.

Reply:
Changed getUniquePVOrPVCName() to getUniqueName() to be more generic since I need to use the method to create a unique configmap name too. Updated all the tests which are using the original method.